CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
+* **New Resource:** `google_clouddeploy_custom_target_type` ([#6956](https://github.com/hashicorp/terraform-provider-google-beta/pull/6956))
+* **New Resource:** `google_network_security_security_profile_group` ([#6961](https://github.com/hashicorp/terraform-provider-google-beta/pull/6961))
+
+IMPROVEMENTS:
+* cloudasset: allowed overriding the billing project for `data.google_cloud_asset_resources_search_all` ([#6941](https://github.com/hashicorp/terraform-provider-google-beta/pull/6941))
+* clouddeploy: added support for `canary_revision_tags`, `prior_revision_tags`, `stable_revision_tags`, and `stable_cutback_duration` to `google_clouddeploy_delivery_pipeline` ([#6951](https://github.com/hashicorp/terraform-provider-google-beta/pull/6951))
+* cloudfunctions: expose `version_id` on `google_cloudfunctions_function` ([#6968](https://github.com/hashicorp/terraform-provider-google-beta/pull/6968))
+* compute: promoted `user_ip_request_headers` field on `google_compute_security_policy` resource to GA ([#6967](https://github.com/hashicorp/terraform-provider-google-beta/pull/6967))
+* container: added support for `enable_relay` field to `advanced_datapath_observability_config` ([#6960](https://github.com/hashicorp/terraform-provider-google-beta/pull/6960))
+* eventarc: added support for `http_endpoint.uri` and `network_config.network_attachment` to `google_eventarc_trigger` ([#6951](https://github.com/hashicorp/terraform-provider-google-beta/pull/6951))
+* healthcare: added `reject_duplicate_message` field to `google_healthcare_hl7_v2_store ` resource ([#6964](https://github.com/hashicorp/terraform-provider-google-beta/pull/6964))
+* identityplatform: added `client`, `permissions`, `monitoring` and `mfa` fields to `google_identity_platform_config` ([#6944](https://github.com/hashicorp/terraform-provider-google-beta/pull/6944))
+* notebooks: made `google_notebooks_instance` wait for active state on creation and enable stopping/starting instances ([#6965](https://github.com/hashicorp/terraform-provider-google-beta/pull/6965))
+* vertexai: added `feature_registry_source` field to `google_vertex_ai_feature_online_store_featureview` resource ([#6962](https://github.com/hashicorp/terraform-provider-google-beta/pull/6962))
+* workbench: added `desired_state` field to `google_workbench_instance` resource ([#6966](https://github.com/hashicorp/terraform-provider-google-beta/pull/6966))
+* workstations: add support for `disable_ssh` in `google_workstations_workstation_config` (beta) ([#6947](https://github.com/hashicorp/terraform-provider-google-beta/pull/6947))
+
+BUG FIXES:
+* compute: fixed doing nothing when `resource_manager_tags` is changed on `google_compute_instance_template` and `google_compute_region_instance_template` ([#6958](https://github.com/hashicorp/terraform-provider-google-beta/pull/6958))
+* notebooks: fixed notebooks instances recreating for scopes and CMEK ([#6948](https://github.com/hashicorp/terraform-provider-google-beta/pull/6948))
```

Comment:
Need fields here, and quotes. Also not entirely sure what this means, can you make it more of a coherent sentence?